SPECS/ocaml-fileutils/ocaml-fileutils.spec

```diff
@@ -35,57 +34,40 @@ abstract filenames.
 
 %package        devel
 Summary:        Development files for %{name}
-Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
 
 %description    devel
 The %{name}-devel package contains libraries and signature files for
 developing applications that use %{name}.
 
 %prep
-%setup -q
-%setup -q -T -a 1 -D
-%autopatch -p1
+%autosetup -n %{name}-%{version} -p1
 
 %build
-ocaml setup.ml -configure --prefix %{_prefix} --destdir %{buildroot} \
-	--enable-tests
-make
+%dune_build
 
 %install
-export DESTDIR=%{buildroot}
-export OCAMLFIND_DESTDIR=%{buildroot}%{_libdir}/ocaml
-mkdir -p $OCAMLFIND_DESTDIR $OCAMLFIND_DESTDIR/stublibs
-
-# Set htmldir to current directory, then copy the docs (in api/)
-# as a %doc rule.
-make htmldir=. install
+%dune_install
 
+# Do not run the tests (RHEL 7+ only) since they require ocaml-ounit.
+%if 0%{?fedora} || 0%{?rhel} || %{?azl} <= 6
 %check
-make test
-
-%files
-%license COPYING.txt
-%{_libdir}/ocaml/fileutils
-%ifarch %{ocaml_native_compiler}
-%exclude %{_libdir}/ocaml/fileutils/*.a
-%exclude %{_libdir}/ocaml/fileutils/*.cmx
-%exclude %{_libdir}/ocaml/fileutils/*.cmxa
-%endif
-%exclude %{_libdir}/ocaml/fileutils/*.ml
-%exclude %{_libdir}/ocaml/fileutils/*.mli
-
-%files devel
-%license COPYING.txt
-%doc AUTHORS.txt CHANGELOG.txt README.txt TODO.txt
-%ifarch %{ocaml_native_compiler}
-%{_libdir}/ocaml/fileutils/*.a
-%{_libdir}/ocaml/fileutils/*.cmx
-%{_libdir}/ocaml/fileutils/*.cmxa
+%dune_check
 %endif
-%{_libdir}/ocaml/fileutils/*.ml
-%{_libdir}/ocaml/fileutils/*.mli
+
+%files -f .ofiles
+%license LICENSE.txt
+
+%files devel -f .ofiles-devel
+%doc README.md CHANGES.md
+%license LICENSE.txt
```

Comment:
Nit-pick: can be dropped. The `devel` package depends on the default one anyway and the default one already contains the license. Not a blocker, though.
```suggestion
```